RevenueCatUI/Data/TestData.swift

```diff
@@ -186,7 +186,8 @@ internal enum TestData {
             colors: .init(light: Self.lightColors, dark: Self.darkColors)
         ),
         localization: Self.localization1,
-        assetBaseURL: Self.paywallAssetBaseURL
+        assetBaseURL: Self.paywallAssetBaseURL,
+        revision: 5
```

Comment:
this quickly became a BIG file. Maybe we should move it to a folder and split into smaller things?

Reply:
it'll only grow over time

Reply:
Yup 👍🏻  I'll do it next week.

Reply:
Also done in #3196